src/Identifier/IdentifierCollection.php

```diff
@@ -28,6 +28,13 @@ class IdentifierCollection extends AbstractCollection implements IdentifierInter
      */
     protected $_errors = [];
 
+    /**
+     * Map of loaded objects.
+     *
+     * @var object[]
+     */
+    protected $_loaded = [];
```

Comment:
Remove this. This class is a dependent of `Cake\Core\ObjectRegistry` which already has the property defined.